pkg/airgapped/plugin_bundle_download.go

```diff
@@ -244,8 +244,12 @@ func (o *DownloadPluginBundleOptions) downloadImagesAsTarFile(pluginEntries []*p
 // validateOptions validates the provided options and returns
 // error if contains invalid option
 func (o *DownloadPluginBundleOptions) validateOptions() error {
-	_, err := os.Stat(filepath.Dir(o.ToTar))
```

Comment:
I would keep this check also, as it allows us to tell the user it is the directory that does not exists.
Without this check we get:
```
$ tz plugin download-bundle --image l --to-tar /bad/dir/plugin.tar
[x] : invalid path for "/bad/dir/plugin.tar": open /bad/dir/plugin.tar: no such file or directory
```
which gives the impression that the file `plugin.tar` must exist first, which is not a requirement.

Reply:
The reason I deleted the existing directory validation is that I have implemented the write-file-validate solution mentioned below.

Reply:
Yes, but even with your improvement if we run

```
$ tz plugin download-bundle --image l --to-tar /bad/dir/plugin.tar
[x] : invalid path for "/bad/dir/plugin.tar": open /bad/dir/plugin.tar: no such file or directory
```

I found that error more confusing that what we have now:
```
[x] : invalid path for "/bad/dir/plugin.tar": stat /bad/dir: no such file or directory
```

What do you think?

Reply:
Agree with you @marckhouzam , added the directory validation.